### PR TITLE
Add Parametric Devices

### PR DESCRIPTION
**Pull request by lmettler**
#### Summary

Add Parametric Devices

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/parametric/rdr-codec.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+uplinkDecoder:
+  fileName: pcr2-decoder.js
+  # Examples (optional)
+  examples:
+    - description: rdr payload
+      input:
+        fPort: 190
+        bytes: [0xbe, 0x05, 0x00, 0x5f, 0x9b, 0xeb, 0x2c, 0x01, 0x00, 0x5f, 0x9b, 0xeb, 0x2c, 0x9e, 0xaf, 0x51, 0x26, 0x00, 0x23, 0x02, 0xe0, 0x01, 0x0c]
+      output:
```
Comment (Jaime-Trinidad) [inline vendor/parametric/rdr-codec.yaml:12]: @lmettler please check decoder file and the output, when you run `make validate` this file is `invalid`

**Comment by lmettler** [comment]
Removed the example from rdr-codec.yaml as it is not possible to match the uplinktime with a string. It references to the time the decoder is executed (with a Date-Object). So everytime the decoder runs, there will be a different uplink time.

By the way, when I validate locally with "make validate", I don't get the error messages you got. Is there another testfile?

**Comment by lmettler** [comment]
@Jaime-Trinidad Sorry for all the failed tests with the payload examples. I didn't get the error messages you got. I commented the examples for now, so it should finally work.

**Comment by Jaime-Trinidad** [comment]
Hi @lmettler please run make validate fmt, I adjust the images that they were causing errors.

**Review by Jaime-Trinidad — APPROVED** [review]

**Pull request merged**